config/routes.rb

```diff
@@ -32,7 +32,9 @@
   # ######## #
   namespace :webhooks do
     resource :stripe, only: [:create], controller: "stripe"
-    resource :paypal, only: [:create], controller: "paypal"
+    resource :paypal, only: [:create], controller: "paypal" do
+      post :ipn
```

Comment:
I've changed the routing pattern. The IPN events will be at `webhooks/paypal/ipn`.
Currently they're at `webhooks/paypal`, but I felt it might make more sense to have `webhooks/paypal` be used for paypal webhook events.